integration-tests/browseSolutionsPage.test.js

```diff
@@ -0,0 +1,27 @@
+import nock from 'nock';

```

Comment:
Could we follow the other file naming by calling this one 'showBrowseSolutionsPage'?